docs/mevblocker/concepts/why-mevblocker.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 1
+---
+
+# Overview
+
+## MEV Blocker is the #1 leading MEV-protection RPC solution by transaction volume and rebates.
+
+MEV Blocker is an RPC endpoint built to:
+
+* Protect users from frontrunning & sandwich attacks
+* Give users a rebate from backrunning opportunities created by the user's trade
+* Provide fast trade settlement
+
+MEV Blocker is built by the experts in MEV supply chain Agnostic Relay, Beaver Build, CoW DAO.
+
+![The Best Protection Under the Sun](/img/mevblocker/mevblocker_second_tag.png)
+
+## Why Integrate MEV Blocker?
+
+* **MEV Protection:** MEV Blocker routes transactions through a private mempool to guard against harmful MEV attacks.
+* **Quick and reliable** Over 90% of transactions are processed in the next block, typically within 12 seconds.
+* **Earn rebates:** MEV Blocker auctions off the right to backrun users' transactions, searchers captures backrunning opportunities and sends 90% back as rebates to transaction originators. MEV Blocker automatically captures backrunning opportunities and sends trade rebates to users (with opportunity for projects to capture such rebates).
+
+## How is MEV Blocker Better?
+
+### Low Latency and Fast Inclusion Time
+
+With a strong builder network, MEV Blocker acts as a builder aggregator where transactions get forwarded directly to network of all the major builders. Over 90% of transactions are processed in the next block, typically within 12 seconds.
+
+### Superior Rebates
+
+With more robust searchers competition, MEV Blocker has been able to generate at least 10x more backrunning rebates than any other competitor solutions with over 1000 ETH rebates to users / projects to date .
+
+### Simple and No $ Cost to Integrate
+
+MEV Blocker is built by experts to be user-friendly, easy to set up, and no $ cost to integrate.
+
+## Growing List of Partners
+
+Top Wallets integrated MEV Blocker include Uniswap Wallet, Rabby Wallet, Crypto.com Defi Wallet, Ambire Wallet, Keepkey Wallet.
```

Comment:
```suggestion
Top Wallets integrating MEV Blocker include Uniswap Wallet, Rabby Wallet, Crypto.com Defi Wallet, Ambire Wallet, Keepkey Wallet.
```

Reply:
Additionally, this is copy-pasta from `README.mdx`. I don't think this should be duplicated.